projects/steering_can/src/steering_can.c

```diff
@@ -0,0 +1,52 @@
+#include "can.h"
+#include "can_fsm.h"
+#include "can_msg.h"
+#include "can_ack.h"
+#include "can_fifo.h"
+#include "can_hw.h"
+#include "can_rx.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "fsm.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "steering_can.h"
+#include "can_transmit.h"
+
+  static CanStorage s_can_storage;
+
+StatusCode steering_can_init() {

```

Comment:
don't init this here, can should be initialized in your main. Multiple modules may want to use can, and not all of them should initialize can every time. Put comments in `steering_can.h` mentioning that this module relies on can being initialized.